kairon/shared/actions/utils.py

```
@@ -1132,4 +1134,24 @@ def trigger_rephrase(bot: Text, llm_type: str, text_response: Text):
+            stack_trace = traceback.format_exc()
+            asyncio.create_task(
+                MailUtility.format_and_send_mail(mail_type="action_failure",
+                                                   email=email,
+                                                   first_name="Team kAIron",
+                                                   stack_trace=stack_trace,
+                                                   slot_values=kwargs.get("slot_values", {}),
+                                                   bot_name=bot_name,
+                                                   action_name=kwargs.get("action_name"),
+                                                   user_query_history=kwargs.get("user_query_history", []),
+                                                   )
+            )
```

Ruff (0.16.1)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)